### PR TITLE
Task 1.3

### PR DESCRIPTION
**Pull request by fujoshik**

### CODE REVIEW

File: Task1.3/Program.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+        var uniqueIntegers = FindUniqueValues(array);
+
+        Console.Write("Unique integers: ");
+        for (int i = 0; i < uniqueIntegers.Length; i++)
+        {
+            Console.Write($"{uniqueIntegers[i]} ");
+        }
+    }
+
```
Comment (IgorKheidorov) [inline Task1.3/Program.cs:24]: Better name is "CreateSet"

File: Task1.3/Program.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    public static int[] FindUniqueValues(int[] array)
+    {
+        int[] uniqueIntegers = new int[array.Length];
+        int uniqueIntegersCount = 0;
+
```
Comment (IgorKheidorov) [inline Task1.3/Program.cs:29]: Seems that it works, but the size of output array has to be = uniqueIntegersCount

**Review by IgorKheidorov — CHANGES_REQUESTED** [review]
Please fix comments

File: Task1.3/Program.cs (hunk 2 above)
Comment (fujoshik) [reply Task1.3/Program.cs:29]: fixed

File: Task1.3/Program.cs (hunk 1 above)
Comment (fujoshik) [reply Task1.3/Program.cs:24]: fixed

**Review by IgorKheidorov — APPROVED** [review]

**Review by IgorKheidorov — APPROVED** [review]

**Pull request merged**